sphinx_needs/api/need.py

```diff
@@ -570,7 +572,7 @@ def add_external_need(
     external_css: str = "external_link",
     content: str = "",
     status: str | None = None,
-    tags: str | None = None,
+    tags: str | list[str] | None = None,
```

Comment:
The docstring of `add_need` and `add_external_need` still says `:param tags: Tags as single string.`

Reply:
fixed